ecl/eclagent/eclagent.ipp

```diff
@@ -954,9 +954,17 @@ private:
         {
             container = _container;
         }
-
+        virtual unsigned getWorkflowId() const override
+        {
+            return wfid;
+        }
+        void setWfid(unsigned _wfid)
+        {
+            wfid = _wfid;
+        }
     protected:
         EclSubGraph * container;
+        unsigned wfid = 0;
```

Comment:
minor: mixture of uninitialized members - I think worth also initializing 'container' at this juncture.